index.js

```diff
@@ -150,9 +136,7 @@ S3Adapter.prototype.uploadFile = function (file, callback) {
 // - the file is set to a canned ACL (ie, headers:{ 'x-amz-acl': 'public-read' } )
 // - you pass credentials during your request for the file content itself
 S3Adapter.prototype.getFileURL = function (file) {
-	// Consider providing an option to use insecure http. I can't think of any
-	// sensible use case for plain http though. https should be used everywhere.
-	return this._knoxForFile(file).https(this._resolveFilename(file));
+	return 'https://' + (file.bucket || this.options.bucket) + '.s3.amazonaws.com' + this._resolveFilename(file);
```

Comment:
Please note in China S3 url's are different. As per  [http://docs.amazonaws.cn/en_us/aws/latest/userguide/s3.html](url)

> The endpoints for services in the Beijing Region are different from other global endpoints. For an endpoint in the Beijing Region, the Region value should be cn-north-1 and the domain should be amazonaws.com.cn. 

In general URL end point is `.s3.cn-north-1.amazonaws.com.cn`